{
  "deprecated": false,
  "cpeName": "cpe:2.3:a:python:setuptools:58.5.1:*:*:*:*:*:*:*",
  "cpeNameId": "954EAAE2-AAB9-4A74-A1C3-A9A3D64DCE85",
  "created": "2022-12-29T14:39:02.883",
  "lastModified": "2022-12-29T14:43:10.977",
  "titles": [
    {
      "title": "Python setuptools 58.5.1",
      "lang": "en"
    }
  ],
  "refs": [
    {
      "ref": "https://pypi.org/project/setuptools/#history",
      "type": "Version"
    },
    {
      "ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
      "type": "Advisory"
    }
  ]
}
